src/config/store.ts

```diff
@@ -0,0 +1,49 @@
+import { configureStore, createAction, CombinedState } from "@reduxjs/toolkit";
+import { combineReducers, Action } from "redux";
+
+import { RESET_STORE_ACTION_TYPE } from "constants/settings";
+import metrics from "middleware/metrics";
+import { routerMiddleware } from "helpers/simpleRouter";
+
+import accountCreator from "reducers/accountCreator";
+import endpointExplorer from "reducers/endpointExplorer";
+import transactionBuilder from "reducers/transactionBuilder";
+import transactionSigner from "reducers/transactionSigner";
+import xdrViewer from "reducers/xdrViewer";
+import network from "reducers/network";
+import routing from "reducers/routing";
+
+export type RootState = ReturnType<typeof store.getState>;
+
+const loggerMiddleware =
+  (storeVal: any) => (next: any) => (action: Action<any>) => {
+    console.log("Dispatching: ", action.type);
+    const dispatchedAction = next(action);
+    console.log("NEW STATE: ", storeVal.getState());
+    return dispatchedAction;
+  };
+
+const reducers = combineReducers({
+  accountCreator,
+  endpointExplorer,
+  transactionBuilder,
+  transactionSigner,
+  xdrViewer,
+  network,
+  routing,
+});
+
+export const resetStoreAction = createAction(RESET_STORE_ACTION_TYPE);

```

Comment:
jc - where do you envision this being used?

Reply:
I thought we'll need something like that when clearing the form (might need to tweak it to exclude network, maybe?). Currently, it clears only the active form, but it should clear everything to avoid manual clearings on each page.

![image](https://user-images.githubusercontent.com/7346473/129090270-282763d2-4431-4048-80f0-fbe11a8a262f.png)